text/0750-deprecate-ember-assign.md

```diff
@@ -0,0 +1,59 @@
+---
+Stage: Accepted
+Start Date: 2021-05-26
+Release Date: Unreleased
+Release Versions:
+  ember-source: vX.Y.Z
+  ember-data: vX.Y.Z
+Relevant Team(s): Ember.js
+RFC PR: https://github.com/emberjs/rfcs/pull/750
+---
```

Comment:
This seems incorrect

Reply:
This is exactly the format specified by the combination of the process [outlined in the README](https://github.com/emberjs/rfcs#the-process) and [specified in the instructions](https://raw.githubusercontent.com/emberjs/rfcs/master/0000-template.md). If that needs tweaking, the instructions in the template are what we should tackle!

Reply:
The format is correct but the information was copied from a different RFC

Reply:
Which information? I keep flipping between this and the template to copy and I’m not getting it

Reply:
Ah. I see why. I was looking at the “Rendered” link which pointed to a specific outdated commit, which had an incorrect header that links to a different RFC issue. Can you fix the link in the PR description? Sorry for the confusion

Reply:
Ahh ok. Fixed